models/article_model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection")
+
+function selectArticle(id) {
+  return db.query(`SELECT * FROM articles WHERE $1 = article_id`, [id])
+    .then((result) => {
+      return result.rows
```

Comment:
lets change this to send back the article object since its only one object inside the array